src/web/app/app.component.ts

```diff
@@ -13,7 +14,10 @@ import { MasqueradeModeService } from '../services/masquerade-mode.service';
 })
 export class AppComponent implements OnInit {
   constructor(private router: Router, private route: ActivatedRoute,
-              private masqueradeModeService: MasqueradeModeService) {}
+              private masqueradeModeService: MasqueradeModeService, tooltipConfig: NgbTooltipConfig) {
+    tooltipConfig.openDelay = 400;
```

Comment:
400ms seems like the standard around the internet though... Felt weird to me at 200ms. Besides, for how many tooltips our website have, I think it is better for a longer delay rather than a shorter one to avoid too many tooltips showing.